qiskit/_compiler.py

```diff
@@ -36,14 +36,15 @@
 from .unroll import DagUnroller, DAGBackend, JsonBackend, Unroller, CircuitBackend
 from .mapper import (Coupling, optimize_1q_gates, coupling_list2dict, swap_mapper,
                      cx_cancellation, direction_mapper)
+from ._gate import Gate
 
 logger = logging.getLogger(__name__)
 
 
 def compile(circuits, backend,
             config=None, basis_gates=None, coupling_map=None, initial_layout=None,
             shots=1024, max_credits=10, seed=None, qobj_id=None, hpc=None,
-            skip_translation=False):
```

Comment:
Do we really want to remove this kwarg?  For backward compatibility, should it not implicitly set `skip_transpiler` and raise a depreciation warning saying that it will be removed in the future?

Reply:
thanks, good point.